cmd/qat_plugin/README.md

```diff
@@ -119,6 +119,28 @@ $ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/
 > socket creation and kubelet registration. Furthermore, the deployments `securityContext` must
 > be configured with appropriate `runAsUser/runAsGroup`.
 
+#### Automatic Provisioning
+
+There's a sample [qat initcontainer](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/build/docker/intel-qat-initcontainer.Dockerfile) that sets cfg_services and enables QAT VFs. To deploy, run as follows:
```

Comment:
the reader may not know what `cfg_services` are. it's also not obvious that SR-IOV can be enabled regardless of what the device is but the services can only be configured for `4xxxvf` with Linux 6.0 being the minimum requirement